src/app/listSubmissions/listSubmissions.lambda.ts

```diff
@@ -11,6 +11,6 @@ export async function handler(event: APIGatewayProxyEventV2): Promise<ApiGateway
     return await requestHandler.handleRequest(params);
   } catch (error: any) {
     console.error(error);
-    return Response.error(400);
+    return Response.error(500);
```

Comment:
Dit is discutabel, maar handleRequest kan ook fout gaan vanwege een client error in eigen code of opbouw natuurlijk. Staat dit voor later op de planning?

Reply:
Oke, maar geen prio en ook logisch nu